libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotificationView.kt

```diff
@@ -0,0 +1,237 @@
+package com.mapbox.navigation.trip.notification
+
+import android.app.PendingIntent
+import android.content.Context
+import android.graphics.Bitmap
+import android.text.SpannableString
+import android.view.View
+import android.widget.RemoteViews
+import androidx.core.content.ContextCompat
+import com.mapbox.navigation.utils.internal.SET_BACKGROUND_COLOR
+
+internal class MapboxTripNotificationView(
+    private val context: Context,
+    private val pendingCloseIntent: PendingIntent?
+) {
+    var collapsedView: RemoteViews? = null
```

Comment:
just curious, why we need `collapsedView` and `expandedView` as `public`(not `private`)? for tests only?

Reply:
I implemented your suggestion:  https://github.com/mapbox/mapbox-navigation-android/pull/4448#discussion_r639564300

Reply:
yeah 😄 why did we need `getCollapsedView()` and `getExpandingView()` **public**? Seems that we these properties(functions) in tests only, right? 
I'd make **private** `collapsedView` and `expandedView` and add a factory for `remoteView`. This way we can mock RemoveViewFactory and hide properties `collapsedView` and `expandedView`. 
Something like:

```kotlin
object RemoteViewFactory {
  fun provideRemoteView(...): RemoteView = ...
}
```

in tests 
```kotlin
@Test // or @Before
fun test1() {
val collapsedView: RemoteView = mockk()
mockObject(RemoteViewFactory)
every {RemoteViewFactory.provideRemoteView(...) } returns mockRemoteView
...
}
```
wdyt?

Reply:
They aren't only accessed in tests. The are accessed in MapboxTripNotification::getNotificationBuilder.